.github/workflows/revised-its.yml

```diff
@@ -36,3 +36,21 @@ jobs:
       script: ./it.sh github ${{ matrix.it }}
       it: ${{ matrix.it }}
       mysql_driver: com.mysql.jdbc.Driver
+
+  query-it:
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: [ 8 ]
+        it: [ Query, QueryRetry, QueryError ]
+        indexer: [ middleManager ]
+    uses: ./.github/workflows/reusable-revised-its.yml
+    env:
```

Comment:
We can't pass env variables from the main workflow to the reusable workflow. It can only be done through inputs currently.

Reply:
Thanks! Have fixed it now.